web/src/app/legal/terms-conditions/terms-conditions.md

```diff
@@ -0,0 +1,46 @@
+**Everything in italics need more attention**
```

Comment:
Headings in the other file having different formatting, e.g. `##` vs `**`, any reason why or can I update it to be the same as the other file?